models/plan.py

```diff
@@ -111,3 +108,26 @@ def close_camps(self, *camps: Camp) -> None:
             raise self.MissingCampsError()
         else:
             self.save()
+
+    def close(self, __is_closed=None):
```

Comment:
What is the argument `__is_closed` for? 
It is not used

Reply:
Removed it now.